share/goodie/ia_hints/title.handlebars

```diff
@@ -0,0 +1 @@
+<span>Pssst! See what happens when you search for </span><strong><a href="https://duckduckgo.com/?q={{sample_query}}">"{{sample_query}}"</a></strong>
```

Comment:
@talsraviv Might want to make this a relative URL so it points to whatever server the code is running on.